lib/tools/system-calls.js

```diff
@@ -438,7 +438,9 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   let proc = new SubProcess(emulatorBinaryPath, launchArgs);
   await proc.start(0);
   proc.on('output', (stdout, stderr) => {
-    log.info(`[AVD OUTPUT] ${stdout || stderr}`);
+    for (let line of (stdout || stderr || '').split('\n').filter(Boolean)) {
```

Comment:
another option would have been to use `lines-stderr` and `lines-stdout` events, which do this for you already. there is however no `lines-output` event which admittedly would be most appropriate for this case.